lib/modules/datasource/gitlab-packages/index.spec.ts

```diff
@@ -8,7 +8,7 @@ describe('modules/datasource/gitlab-packages/index', () => {
     it('returns package from custom registry', async () => {
       const body = [
         {
-          version: '1.0.0',
+          version: 'v1.0.0',
```

Comment:
Added this `v` to match the others below.

Reply:
Are you sure it wasn't present to test absence of v prefix?

Reply:
Fair point, reverting this.